signalfx-transform-helper.js

```diff
@@ -20,6 +20,8 @@ function flatten(obj, newObj, prefix) {
   for (let [key, value] of Object.entries(obj)) {
     if (isPrimitive(key) && isPrimitive(value)) {
       newObj[sanitize(prefix + key)] = value;
+    } else if (prefix !== ''){
+      newObj[sanitize(prefix + key)] = JSON.stringify(value);
```

Comment:
I believe we should basically JSON.stringify(value) any non-primitive value without any prefix.

Reply:
yep, now works as you describe